packages/kotti-ui/source/kotti-i18n/locales/de-DE.ts

```diff
@@ -27,6 +27,48 @@ export const deDE: KottiI18n.Messages = {
 		replyButton: 'Antworten',
 		replyPlaceholder: 'Antwort an',
 	},
+	KtFieldFileUpload: {
+		button: {
+			acceptPhoto: 'Foto verwenden',
+			cancel: 'Absagen',
+			nextCamera: 'Nächste Kamera',
+			rejectPhoto: 'Foto wiederholen',
+			retry: 'Wiederholen',
+			takePhoto: 'Foto machen',
+		},
+		error: {
+			notAllowed: 'Die Erlaubnis zur Verwendung der Kamera wurde verweigert',
+			notFound: 'Keine verfügbare Kamera gefunden',
+			notSupported: 'Keine kompatible Kamera verfügbar',
+		},
+		label: {
+			capture: 'Foto machen',
+			error: 'Error',
+			review: 'Überprüfen',
+			unknown: 'Unbekannt',
+		},
+		statusMsg: {
+			CANCELED: 'Abgebrochen',
```

Comment:
Minor

this is english-us - I'm not sure if we have a convention for english-uk vs english-us

I personally don't care and didn't even find other instances of cancelled

but if you know a word that distinguishes US from UK english and you see that kotti has a tendency to use either, follow the one it follows

Reply:
Didn't find anything about which is preferred in readme. Assumed it is en-us as 'gray' is used instead of 'grey'. There's also this commit f992d3a

Reply:
@carsoli convention is using en-US, we even had fixes for accidental UK English use in the past